test/webclient/cypress/integration/roller_15_raStatements_spec.js

```diff
@@ -6,12 +6,14 @@ import * as common from '../support/utils/common';
 
 // --- Collections ---
 const section = require('../support/components/raStatements'); // RA Statements
+const section2 = require('../support/components/raStatements2'); // RA Statements
```

Comment:
Provide proper section name instead section2. Do same changes in another file as well.